iron-input.html

```diff
@@ -183,6 +188,7 @@
       if (this.preventInvalidInput && !this._patternAlreadyChecked) {
         var valid = this._checkPatternValidity();
         if (!valid) {
+          this._announceInvalidCharacter('Invalid string of characters not entered.');
```

Comment:
a) re: "The string..."
b) I wrote in the description why we don't know exactly what the string you entered was. Let me know if it makes sense.